internal/cmd/backup.go

```diff
@@ -118,6 +120,39 @@ func runBackup(c *cli.Context) error {
 		}
 	}
 
+	// Repositories exclude mirror repositories
+	if !c.Bool("exclude-mirror-repos") && !c.Bool("database-only") {
+		reposFile := filepath.Join(rootDir, "non-mirror-repositories.zip")
```

Comment:
We have convention to use "repositories.zip" in `./gogs restore` command, so please use that.